src/utils/legacy-settings.js

```diff
@@ -130,12 +130,21 @@ function deserializeSettingForLegacy(data, settingId) {
       const hideOfflineFollowedChannels = data[LegacySettingIds.HIDE_OFFLINE_FOLLOWED_CHANNELS] || false;
       const autoExpand = data[LegacySettingIds.AUTO_EXPAND_CHANNELS] || false;
 
-      let flags = setFlag(0, SidebarFlags.FEATURED_CHANNELS, !hideFeaturedChannels);
+      let flags = setFlag(
+        SidebarFlags.FEATURED_CHANNELS,
+        SidebarFlags.RECOMMENDED_CHANNELS | SidebarFlags.SIMILAR_CHANNELS,
+        !hideFeaturedChannels
+      );
       flags = setFlag(flags, SidebarFlags.OFFLINE_FOLLOWED_CHANNELS, !hideOfflineFollowedChannels);
       flags = setFlag(flags, SidebarFlags.AUTO_EXPAND_CHANNELS, autoExpand);
       return [
         flags,
-        SidebarFlags.FEATURED_CHANNELS | SidebarFlags.OFFLINE_FOLLOWED_CHANNELS | SidebarFlags.AUTO_EXPAND_CHANNELS,
+        SidebarFlags.FEATURED_CHANNELS |
```

Comment:
this looks to change the behavior of the import. I think you want:

```
SidebarFlags.FEATURED_CHANNELS |
SidebarFlags.OFFLINE_FOLLOWED_CHANNELS |
SidebarFlags.AUTO_EXPAND_CHANNELS |
SidebarFlags.RECOMMENDED_CHANNELS |
SidebarFlags.SIMILAR_CHANNELS
```

these flags are bits that are changed by the user. i.e. the user toggled these settings at least once.

Reply:
done 👍